### DISCUSSION TITLE
Geomap: Add "route" to "legend"

### DISCUSSION THREAD

**Discussion by aOk88** [grafana/grafana#72640] [Dashboards, Panels & Visualizations]
Currently only markers supported in legend

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
